### PR TITLE
Add ElasticBox KB Edits DLM

### PR DESCRIPTION
**Pull request by DLM1**
file path clean up mainly, a couple of spelling errors

### CODE REVIEW

File: ElasticBox/appliance-openstack.md [hunk 1]
```diff
@@ -7,12 +7,30 @@
 }}}
 
 ### Installing the Virtual Appliance in OpenStack
+<<<<<<< HEAD
```
Comment (srveit) [inline ElasticBox/appliance-openstack.md]: Clean up this section. remove the `<<<<<<< HEAD`, `=======`, and `>>>>>>> CenturyLinkCloud/master` lines. Also remove the duplicate content.

File: ElasticBox/admin-access.md [hunk 2]
```diff
@@ -18,10 +18,24 @@ When you sign up for the ElasticBox Enterprise Edition, you get a default admini
    ![admin-access2.png](../images/ElasticBox/admin-access2.png)
 
 To revoke administrator privileges for a user, click X against their name under **Organization > Administrators**.
+<<<<<<< HEAD
```
Comment (srveit) [inline ElasticBox/admin-access.md]: Clean up this section. remove the `<<<<<<< HEAD`, `=======`, and `>>>>>>> CenturyLinkCloud/master` lines. Also remove the duplicate content.

File: ElasticBox/admin-access.md [hunk 3]
```diff
@@ -18,10 +18,24 @@ When you sign up for the ElasticBox Enterprise Edition, you get a default admini
...
+>>>>>>> CenturyLinkCloud/master
+=======
 Administrators can manage several org-wide settings that include [user authentication](../ElasticBox/user-authentication.md), the [ElasticBox appliance](../ElasticBox/appliance-overview.md), [provider access](../ElasticBox/provider-access.md), [admin boxes](../ElasticBox/deploymentpolicy-box.md), [webhooks](../ElasticBox/webhooks.md), and [instance tags](../ElasticBox/resource-tags.md). To get a good idea of things you can do, see [Overview](../ElasticBox/admin-overview.md).
 
 ### Contacting ElasticBox Support
 We’re sorry you’re having an issue in [ElasticBox](//www.ctl.io/elasticbox/). Please review the [troubleshooting tips](https://elasticbox.com/documentation/troubleshooting/troubleshooting-tips/), or contact [ElasticBox support](mailto:support@elasticbox.com) with details and screen shots where possible.
+>>>>>>> CenturyLinkCloud/master
```
Comment (srveit) [inline ElasticBox/admin-access.md]: Remove this line.

File: ElasticBox/appliance-openstack.md [hunk 4]
```diff
@@ -36,11 +54,25 @@ Follow the video for detailed help.
 3. Launch an instance of ElasticBox from the appliance image and flavor.
 
 ### Next Steps
+<<<<<<< HEAD
```
Comment (srveit) [inline ElasticBox/appliance-openstack.md]: Clean up this section. remove the `<<<<<<< HEAD`, `=======`, and `>>>>>>> CenturyLinkCloud/master` lines. Also remove the duplicate content.

File: ElasticBox/appliance-overview.md [hunk 5]
```diff
@@ -7,15 +7,43 @@
 }}}
 
 ### ElasticBox Enterprise Appliance Overview
+<<<<<<< HEAD
```
Comment (srveit) [inline ElasticBox/appliance-overview.md]: Clean up this section. remove the `<<<<<<< HEAD`, `=======`, and `>>>>>>> CenturyLinkCloud/master` lines. Also remove the duplicate content.

File: ElasticBox/appliance-overview.md [hunk 6]
```diff
@@ -7,15 +7,43 @@
...
 
 **Try the Appliance for Free**
 The appliance is available as a free download. Experience all the same functionality as the paid Enterprise Edition, except for advanced features like managing your ElasticBox organization, sharing and monitoring resources, and deploying to private clouds. To get these features, upgrade to the Enterprise Edition. To learn more, see [ElasticBox Editions](../ElasticBox/about-elasticbox-editions.md).
 
+>>>>>>> CenturyLinkCloud/master
 **Download the ElasticBox Virtual Appliance**
 Request a download link from [support](mailto:support@elasticbox.com).
 
 ### Next Up
+<<<<<<< HEAD
```
Comment (srveit) [inline ElasticBox/appliance-overview.md]: Clean up this section. remove the `<<<<<<< HEAD`, `=======`, and `>>>>>>> CenturyLinkCloud/master` lines. Also remove the duplicate content.

File: ElasticBox/appliance-vsphere.md [hunk 7]
```diff
@@ -10,10 +10,23 @@
 If you don’t have internet access in the vCenter, download all the files and import them locally into your vCenter network. If you have access, you can directly import the appliance in the open virtualization format (OVF) and vCenter will download the disk images for you.
 
 **In this article:**
+<<<<<<< HEAD
```
Comment (srveit) [inline ElasticBox/appliance-vsphere.md]: Clean up this section. remove the `<<<<<<< HEAD`, `=======`, and `>>>>>>> CenturyLinkCloud/master` lines. Also remove the duplicate content.

File: ElasticBox/appliance-vsphere.md [hunk 8]
```diff
@@ -50,11 +66,25 @@ In these steps, you connect the ElasticBox appliance to the vCenter datacenter n
 3. Now start the appliance. Right-click the appliance, click **Power > Power On**.
 
 ### Next Steps
+<<<<<<< HEAD
```
Comment (srveit) [inline ElasticBox/appliance-vsphere.md]: Clean up this section. remove the `<<<<<<< HEAD`, `=======`, and `>>>>>>> CenturyLinkCloud/master` lines. Also remove the duplicate content.

File: ElasticBox/reporting.md [hunk 9]
```diff
@@ -13,9 +13,21 @@ Reports show cloud costs and resource usage in public and private clouds. It inc
 
 If reporting is turned on for your enterprise, ElasticBox Administrators can access it from the admin console. Reports also help you make budget-friendly decisions and curb costs through cost center budgeting.
 
+<<<<<<< HEAD
```
Comment (srveit) [inline ElasticBox/reporting.md]: Clean up this section. remove the `<<<<<<< HEAD`, `=======`, and `>>>>>>> CenturyLinkCloud/master` lines. Also remove the duplicate content.

File: ElasticBox/reporting.md [hunk 10]
```diff
@@ -112,7 +124,15 @@ Cost Centers help you set and enforce a quota on how much users or team workspac
    ![admin-reports14.png](../images/ElasticBox/admin-reports14.png)
 
 ### Contacting ElasticBox Support
+<<<<<<< HEAD
```
Comment (srveit) [inline ElasticBox/reporting.md]: Clean up this section. remove the `<<<<<<< HEAD`, `=======`, and `>>>>>>> CenturyLinkCloud/master` lines. Also remove the duplicate content.

File: ElasticBox/user-authentication.md [hunk 11]
```diff
@@ -7,12 +7,30 @@
 }}}
 
 ### Enable User Authentication
+<<<<<<< HEAD
```
Comment (srveit) [inline ElasticBox/user-authentication.md]: Clean up this section. remove the `<<<<<<< HEAD`, `=======`, and `>>>>>>> CenturyLinkCloud/master` lines. Also remove the duplicate content.

File: ElasticBox/user-authentication.md [hunk 12]
```diff
@@ -27,17 +45,43 @@ Add LDAP sources in ElasticBox to match the structure of LDAP in your organizati
 When users sign in to ElasticBox with their LDAP credentials, we don’t store their passwords. The login session passes on their credentials to each LDAP source defined in ElasticBox. The LDAP server looks the user up by their username or Use Principal Name (UPN), typically in the **yourname@example.com** format. The server responds with an authorized or unauthorized request. If authorized, we grant the user access in ElasticBox. Else, we deny access.
 
 **In this article:**
+<<<<<<< HEAD
```
Comment (srveit) [inline ElasticBox/user-authentication.md]: Clean up this section. remove the `<<<<<<< HEAD`, `=======`, and `>>>>>>> CenturyLinkCloud/master` lines. Also remove the duplicate content.

File: ElasticBox/user-authentication.md [hunk 13]
```diff
@@ -27,17 +45,43 @@ Add LDAP sources in ElasticBox to match the structure of LDAP in your organizati
...
+=======
 * [Setting up LDAP in ElasticBox](../ElasticBox/user-authentication.md)
 * [Syncing with LDAP groups](../ElasticBox/user-authentication.md)
 * [Giving LDAP accounts admin access](../ElasticBox/user-authentication.md)
+>>>>>>> CenturyLinkCloud/master
 
 ### Setting Up LDAP in ElasticBox
 
 **Steps**
+<<<<<<< HEAD
```
Comment (srveit) [inline ElasticBox/user-authentication.md]: Clean up this section. remove the `<<<<<<< HEAD`, `=======`, and `>>>>>>> CenturyLinkCloud/master` lines. Also remove the duplicate content.

File: ElasticBox/user-authentication.md [hunk 14]
```diff
@@ -71,9 +115,21 @@ LDAP groups get automatic access to team workspaces in ElasticBox when you enabl
 This gives your developers, operations engineers, or IT admins access to the same deployment assets to do their part in automating with necessary access levels. Follow these steps to sync with LDAP groups.
 
 **Steps**
+<<<<<<< HEAD
```
Comment (srveit) [inline ElasticBox/user-authentication.md]: Clean up this section. remove the `<<<<<<< HEAD`, `=======`, and `>>>>>>> CenturyLinkCloud/master` lines. Also remove the duplicate content.

File: ElasticBox/user-authentication.md [hunk 15]
```diff
@@ -89,7 +145,15 @@ As good practice, you should give an LDAP user in your organization administrati
 3. Make the LDAP user an administrator. From here on, use that LDAP user account to manage ElasticBox.
 
 ### Contacting ElasticBox Support
+<<<<<<< HEAD
```
Comment (srveit) [inline ElasticBox/user-authentication.md]: Clean up this section. remove the `<<<<<<< HEAD`, `=======`, and `>>>>>>> CenturyLinkCloud/master` lines. Also remove the duplicate content.

File: ElasticBox/user-authentication.md (hunk 13 above)
Comment (DLM1) [inline ElasticBox/user-authentication.md]: done

File: ElasticBox/user-authentication.md (hunk 15 above)
Comment (DLM1) [inline ElasticBox/user-authentication.md]: done

File: ElasticBox/user-authentication.md (hunk 14 above)
Comment (DLM1) [inline ElasticBox/user-authentication.md]: done

File: ElasticBox/user-authentication.md (hunk 12 above)
Comment (DLM1) [inline ElasticBox/user-authentication.md]: done

File: ElasticBox/user-authentication.md (hunk 11 above)
Comment (DLM1) [inline ElasticBox/user-authentication.md]: done

File: ElasticBox/reporting.md (hunk 10 above)
Comment (DLM1) [inline ElasticBox/reporting.md]: done

File: ElasticBox/reporting.md (hunk 9 above)
Comment (DLM1) [inline ElasticBox/reporting.md]: done

File: ElasticBox/appliance-vsphere.md (hunk 7 above)
Comment (DLM1) [inline ElasticBox/appliance-vsphere.md]: done

File: ElasticBox/appliance-vsphere.md (hunk 8 above)
Comment (DLM1) [inline ElasticBox/appliance-vsphere.md]: done

File: ElasticBox/appliance-overview.md (hunk 5 above)
Comment (DLM1) [inline ElasticBox/appliance-overview.md]: done

File: ElasticBox/appliance-overview.md (hunk 6 above)
Comment (DLM1) [inline ElasticBox/appliance-overview.md]: done

File: ElasticBox/appliance-openstack.md (hunk 4 above)
Comment (DLM1) [inline ElasticBox/appliance-openstack.md]: done

File: ElasticBox/admin-access.md (hunk 3 above)
Comment (DLM1) [inline ElasticBox/admin-access.md]: done

File: ElasticBox/appliance-openstack.md (hunk 1 above)
Comment (DLM1) [inline ElasticBox/appliance-openstack.md]: done

File: ElasticBox/admin-access.md (hunk 2 above)
Comment (DLM1) [inline ElasticBox/admin-access.md]: done

**Comment by DLM1** [comment]
OK. fixed the 2 files above.

**Comment by srveit** [comment]
Closing this as a duplicate of https://github.com/CenturyLinkCloud/PublicKB/pull/1436

**Pull request closed without merging**